packages/preview-middleware-client/src/adp/quick-actions/fe-v4/enable-variant-management.ts

```diff
@@ -0,0 +1,86 @@
+import type FlexCommand from 'sap/ui/rta/command/FlexCommand';
+
+import { QuickActionContext, SimpleQuickActionDefinition } from '../../../cpe/quick-actions/quick-action-definition';
+
+import { SimpleQuickActionDefinitionBase } from '../simple-quick-action-base';
+import Component from 'sap/ui/core/Component';
+import { getUi5Version, isLowerThanMinimalUi5Version } from '../../../utils/version';
+import { createManifestPropertyChange } from '../../../utils/fe-v4';
+import ListReportComponent from 'sap/suite/ui/generic/template/ListReport';
```

Comment:
This is FE v2 component. It shouldn't be used in FE v4.

Reply:
fixed